### ISSUE TITLE
Queue Time Not Visible

### ISSUE THREAD

**Issue by Zakkumaru** [TeamNewPipe/NewPipe#6254]
### Checklist

- [x] I am using the latest version - x.xx.x 
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Steps to reproduce the bug

Basically, just queuing songs and looking at the queue.

### Actual behaviour

No queue time shows up

### Expected behavior

The queue time to be visible, per #6023 included in the latest release.

### Device info

 - Android version/Custom ROM version:  LineageOS 17.1

**Comment by tsiflimagas** [comment]
Is this still an issue?

**Comment by SameenAhnaf** [comment]
@tsiflimagas Yes, queue time is missing for popup and background queue. It's available for main queue although. Queue time could be added in the marked area.

![IMG_20210913_085055](https://user-images.githubusercontent.com/79245641/133016826-d761c3fa-4772-4dc2-ae35-01212505e7ff.jpg)

**Comment by SameenAhnaf** [comment]
@TippyLion28 #6045 is there for that.
